.travis.yml

```diff
@@ -68,207 +63,48 @@ addons:
-  - stage: build project 🔨
-    if: (type = pull_request) AND (commit_message =~ /Travis-CI-Disable/)
-    name: "Diff Control"
-    script:
-    # Fails if there's any source file modified.
-    - git diff $(git merge-base master HEAD) HEAD --stat --exit-code -- --path "*.hs"
-    - git diff $(git merge-base master HEAD) HEAD --stat --exit-code -- --path "*.js"
-    - git diff $(git merge-base master HEAD) HEAD --stat --exit-code -- --path "specifications/*"
-
-  - stage: build project 🔨
-    if: (type = pull_request) AND NOT (commit_message =~ /Travis-CI-Disable/)
-    name: "Compiling Project"
-    script:
-    - stack clean
-    - stack --no-terminal build --fast --test --no-run-tests --bench --no-run-benchmarks --haddock --no-haddock-deps
-    - tar czf $STACK_WORK_CACHE .stack-work lib/**/.stack-work
-
-  - stage: checks 🔬
-    if: (type = pull_request) AND NOT (commit_message =~ /Travis-CI-Disable/)
-    name: "Code Style"
-    script:
-    - travis_retry curl -sSL https://raw.github.com/jaspervdj/stylish-haskell/master/scripts/latest.sh | sh -s $(find . -type f -name "*.hs" ! -path "*.stack-work*") -i
-    - git diff --exit-code
-    - travis_retry curl -sSL https://raw.github.com/ndmitchell/hlint/master/misc/run.sh | sh -s .
-    - travis_retry curl -sSL https://raw.github.com/ndmitchell/weeder/master/misc/travis.sh | sh -s .
-    # Somehow, we can't install open-spec-validator using `pip` directly in the travis VM.
-    # I've tried several things, including removing some existing conflicting packages, upgrading
-    # pip and using some more permissive options. In the end, I gave up and went for a container.
-    - cd specifications/api
-    - "echo -n 'FROM python:3.7-alpine\nRUN pip install openapi-spec-validator\nENTRYPOINT [\"openapi-spec-validator\"]' > Dockerfile"
-    - docker build -t openapi-spec-validator .
-    - docker run -v $PWD:/files openapi-spec-validator --schema 2.0 /files/swagger.yaml
-    - cd -
-
-  - stage: checks 🔬
-    if: (type = pull_request) AND NOT (commit_message =~ /Travis-CI-Disable/)
-    name: "Tests: unit"
-    script:
-      - stack --no-terminal test --fast cardano-wallet-core cardano-wallet-launcher cardano-wallet-cli text-class bech32 cardano-wallet-http-bridge:unit cardano-wallet-jormungandr:unit
-
-  ################################################################################
-  #
-  # Push / Cron / Api
-  #
-  # This runs for other type of build (non PR) and are slightly heavier since they
-  # also compute coverage and documentation. This is also the only way to rebuild
-  # an initial default cache and re-inject 3rd-party dependencies like:
-  #
-  # - cardano-node-simple
-  # - cardano-http-bridge
-  # - jormungandr
-  # - jcli
-  #
-  ################################################################################
   - stage: cache 💾
-    if: (type != pull_request AND branch = master)
+    if: type != pull_request AND (branch = master OR tag =~ ^v)
```

Comment:
Why is the `OR tag =~ ^v` needed now?

Reply:
I guess because we didn't need the cache when deploying, previously.

Reply:
Well, before, if we had no cache, the release build would just fail and crash, which we never ran into... Now I am just making sure that it's safe.